Doc/howto/logging.rst

```diff
@@ -382,6 +382,10 @@ Logging Flow
 The flow of log event information in loggers and handlers is illustrated in the
 following diagram.
 
+.. only:: latex
+
+   .. image:: logging_flow.png
```

Comment:
IMO we want to:
- use this all builders except `html`
- search for a `logging_flow` image with any extension/format

```suggestion
.. only:: not html

   .. image:: logging_flow.*
```